### ISSUE TITLE
Treatment Supporter Nulls

### ISSUE THREAD

**Issue by thnesh** [chap-uzima/kenyaemrMigration#2]
DML_HIV_Enrollment_v1.0 Returns Nulls in all Treatment supporter Details

**Comment by thnesh** [comment]
The where clause in lines 135 to 138 are responsible for this
WHERE (
			lt.Name = 'TreatmentSupporter'
			OR pts.ContactCategory = 1
			)
Also only the Phone number is being decrypted
The contactcategory and contactrelations in my test DB are zero

**Comment by thnesh** [comment]
@vogoti @mkaburu The FisrtName, MiddleName & LastName in my Persion View where the treatment supporter names are being pulled from are Nulls. Check if same for other DBs

**Comment by mkaburu** [comment]
@thnesh Yes its all NULLS . I think the problem could be in the decryption

**Comment by vogoti** [comment]
> @vogoti @mkaburu The FisrtName, MiddleName & LastName in my Person View where the treatment supporter names are being pulled from are Nulls. Check if same for other DBs. (Line 124 & 129)

The view fails to decrypt the names returning nulls,if you remove the decryption it populates

**Comment by vogoti** [comment]
> The where clause in lines 135 to 138 are responsible for this
> WHERE (
> lt.Name = 'TreatmentSupporter'
> OR pts.ContactCategory = 1
> )
> Also only the Phone number is being decrypted
> The contactcategory and contactrelations in my test DB are zero

The Patient Category in PatientTreatmentSupporter is zero for all clients

**Comment by vogoti** [comment]
Test Slack

**Comment by vogoti** [comment]
Treatment Supporter Address also returns NULL because of Decryption in PersonContactView

**Comment by vogoti** [comment]
Relationship_of_treatment_supporter seems Ok for me,returns NULL incase the Contact 
Person's relationship is not documented

**Comment by mkaburu** [comment]
When you change the ContactCategory = 0 the query should display if we have a treatment supporter documented. Using Mathari DB which has good documentation of treatment supporter details the data is populating well
